src/components/ButtonPanel.js

```diff
@@ -0,0 +1,37 @@
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div>
+    <div className="btn-group">
+      <Button buttonName="AC" />
+      <Button buttonName="+/-" />
+      <Button buttonName="%" />
+      <Button buttonName="÷" />
+    </div>
+    <div className="btn-group">
+      <Button buttonName="7" />
+      <Button buttonName="8" />
+      <Button buttonName="9" />
+      <Button buttonName="X" />
+    </div>
+    <div className="btn-group">
+      <Button buttonName="4" />
+      <Button buttonName="5" />
+      <Button buttonName="6" />
+      <Button buttonName="-" />
+    </div>
+    <div className="btn-group">
+      <Button buttonName="1" />
+      <Button buttonName="2" />
+      <Button buttonName="3" />
+      <Button buttonName="+" />
+    </div>
+    <div className="btn-group">
+      <Button buttonName="0" />
+      <Button buttonName="." />
+      <Button buttonName="=" />
+    </div>
+  </div>
```

Comment:
let's practice [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code.

Instead of repeating ourselves, let's group the signs and numbers as stated in the project requirement;

![image](https://user-images.githubusercontent.com/31897434/104459588-3b729400-55ad-11eb-9ae1-09ae11eb7b68.png)

https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376016/